ProteoformSuiteInternal/Results.cs

```diff
@@ -0,0 +1,434 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
I deleted this class, since it took so much upkeep. Hoping to revise how we save progress with XML structures.